src/test/java/utils/example/utils/MyArrayListTest.java

```diff
@@ -11,21 +11,42 @@
     public void addListTest() {
         MyArrayList expectedEmployeeData = new MyArrayList();
         expectedEmployeeData.add("Muhammad");
-        expectedEmployeeData.add("Ahmed");
-        expectedEmployeeData.add("abdullah");
+        expectedEmployeeData.add(1);
+        expectedEmployeeData.add(33.5);
 
         MyArrayList employee = new MyArrayList();
         assertNotNull(expectedEmployeeData);
         assertEquals(employee.equals(expectedEmployeeData), expectedEmployeeData.equals(employee));
     }
 
     @Test
-    public void getIndexTest(){
+    public void getIndexTest() throws Exception {
         MyArrayList employee = new MyArrayList();
```

Comment:
Fix this test, all of them. Create MyArrayList objects with generics. like ArrayList<Object> etc